sdk/highlight-node/src/sdk.ts

```diff
@@ -25,7 +25,8 @@ export interface HighlightInterface {
 		tags?: { name: string; value: string }[],
 	) => void
 	flush: () => Promise<void>
-	log: (...data: any[]) => void
+	log: (message: any, level: string, ...optionalParams: any[]) => void
```

Comment:
What do you think about exposing this `level` as a string union?
https://github.com/highlight/highlight/blob/702bc392acb78af5416bd06584d1db87e139aae0/sdk/highlight-node/src/hooks.ts#L37-L47

Something like:

```ts
const Levels = {
		debug: 'debug',
		info: 'info',
		log: 'info',
		count: 'info',
		dir: 'info',
		warn: 'warn',
		assert: 'warn',
		error: 'error',
		trace: 'trace',
	} as { [k in keyof Console]: string }

type Level = keyof Levels
```

And replacing `level: string` with `level: Level`